test/case/ietf_routing/ospf_default_route_advertise/test.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env python3
+"""
+OSPF Default route advertise
+
```

Comment:
I think it can be helpful to understand where the OSPF area ends, either in the graphviz picture on with some ascii-art. And perhaps indicate where R1 will have its default route.
I do not understand the purpose of the link 'PC:data -- R2:data'. Should it be removed if it does not add anything to the test?